setup/cli.php

```diff
@@ -67,9 +67,13 @@ function build_container_for_setup(string $executed_in_directory)
             $c["command.install"],
             $c["command.update"],
             $c["command.build-artifacts"],
-            $c["command.reload-control-structure"]
+            $c["command.reload-control-structure"],
+            $c["command.replace-logo"]
```

Comment:
@thibsy not sure if this need an own command like this, maybe only a Objective would be sufficient?

Reply:
Would be sufficient, I implemented a command to simplify the composer.json entry. However, something like this should also be possible: `php -r "(new ReplaceResponsiveLogoObjective())->execute(new ArrayEnvironment([]))"`.